src/ecstatic/runners/TAJSRunner.py

```diff
@@ -0,0 +1,74 @@
+import os.path
+from typing import List, Dict
+
+from src.ecstatic.models.Level import Level
+from src.ecstatic.models.Option import Option
+from src.ecstatic.runners.CommandLineToolRunner import CommandLineToolRunner
+from src.ecstatic.util.UtilClasses import BenchmarkRecord
+
+
+class TAJSRunner (CommandLineToolRunner):
+    def get_timeout_option(self) -> List[str]:
+        if self.timeout is None:
+            return []
+        else:
+            return f"-time-limit {self.timeout * 60}".split(" ")
+
+    def get_whole_program(self) -> List[str]:
+        # shouldn't be necessary
+        return []
+
+    def get_input_option(self, benchmark_record: BenchmarkRecord) -> List[str]:
+        return f" {benchmark_record.name}".split()
+
+    def get_output_option(self, output_file: str) -> List[str]:
+        # callgraph recieves file path argument
+        return f"-callgraph {output_file}".split()
+        
+    def get_task_option(self, task: str) -> List[str]:
+        # might need to add to this
+        if task == 'cg':
+            return []
+        else:
+            raise NotImplementedError(f'TAJS does not support task {task}.')
+
+    def dict_to_config_str(self, config_as_dict: Dict[Option, Level]) -> str:
+        """
+        We need special handling of TAJS's options, because of unsound options and commands without level value
+
+
+        Parameters
+        ----------
+        config_as_dict: The dictionary specifying the configuration.
+        Returns
+        -------
+        The corresponding command-line string.
+        """
+        config_as_str = ""
+        for k, v in config_as_dict.items():
+            k: Option
+            v: Level
+            for t in k.tags:
+                if t.startswith('unsound'):
+                    config_as_str = config_as_str + f"-unsound -{k.name} "
```

Comment:
You would know better than me but can you verify that the syntax to pass an unsoundness option named "foo" is `-unsound -foo` and not `-unsound foo`?

Reply:
I just ran TAJS again-- the syntax is `-unsound -foo`, and it does not work without the dash on the second option.